### PR TITLE
Added info on managing Organization members, API tokens, Agent tokens

### PR DESCRIPTION
**Pull request by karensawrey**

### CODE REVIEW

File: pages/pipelines/permissions.md.erb [hunk 1]
```diff
@@ -60,10 +60,43 @@ When you enable Teams we’ll create a default team called “Everyone”, conta
...
 #### Can I delete the “Everyone” team?
-Yes you can delete or edit the “Everyone” team. To ensure uninterrupted access to pipelines we recommend creating new teams before deleting the “Everyone” team.
+Yes, you can delete or edit the “Everyone” team. To ensure uninterrupted access to pipelines we recommend creating new teams before deleting the “Everyone” team.
 
 #### Once enabled, can I disable Teams?
-You can disable teams by deleting all your teams, and then selecting “Disable Teams”.
+Yes, you can disable teams by deleting all your teams, and then selecting “Disable Teams”.  
+
+#### Can I automate the removal of users from Buildkite?  
+Yes, you can do it using the GraphQL API (you can use [GraphQL Explorer console](https://buildkite.com/user/graphql/console) or [Buildkite GraphQL Explorer](https://graphql.buildkite.com/explorer) if you have configured your Buildkite API Token).
```
Comment (plaindocs) [inline pages/pipelines/permissions.md.erb:69]: Even thought we're using the Explorer to confirm how the method works, if they want to automate it they can't really use the Explorer... How about

```suggestion
Yes, you can automatically remove users using the GraphQL API. You'll need a [GraphQL API token](https://buildkite.com/user/api-access-tokens).
```

File: pages/pipelines/permissions.md.erb [hunk 2]
```diff
@@ -60,10 +60,43 @@ When you enable Teams we’ll create a default team called “Everyone”, conta
...
+Yes, you can delete or edit the “Everyone” team. To ensure uninterrupted access to pipelines we recommend creating new teams before deleting the “Everyone” team.
 
 #### Once enabled, can I disable Teams?
-You can disable teams by deleting all your teams, and then selecting “Disable Teams”.
+Yes, you can disable teams by deleting all your teams, and then selecting “Disable Teams”.  
+
+#### Can I automate the removal of users from Buildkite?  
+Yes, you can do it using the GraphQL API (you can use [GraphQL Explorer console](https://buildkite.com/user/graphql/console) or [Buildkite GraphQL Explorer](https://graphql.buildkite.com/explorer) if you have configured your Buildkite API Token).  
+You'll need to look up your organization's slug in the [Organization Settings](https://buildkite.com/organizations/-/settings) and check the name of the user you want to remove in the [team](https://buildkite.com/organizations/-/teams) that this user belongs to. Knowing this information, run the following script to get the ID of the user as you will need it for the removal process:
```
Comment (plaindocs) [inline pages/pipelines/permissions.md.erb:70]: We may as well tell them explicitly to replace 

your-organization-slug
Jane Doe -> user name or email

(Email works also, and prob makes more sense)

File: pages/pipelines/permissions.md.erb [hunk 3]
```diff
@@ -60,10 +60,43 @@ When you enable Teams we’ll create a default team called “Everyone”, conta
...
+            name
+            email
+          }
+        }
+      }
+    }
+  }
+}
+```  
+
```
Comment (plaindocs) [inline pages/pipelines/permissions.md.erb:89]: And if we're moving away from talking about the GraphQL console, and just talking about the GraphQL API, we could say something like use the first query to get the user ID, and then run the RemoveOrganizationMember mutation with the user ID to delete the user.

File: pages/pipelines/permissions.md.erb [hunk 4]
```diff
@@ -60,10 +60,43 @@ When you enable Teams we’ll create a default team called “Everyone”, conta
...
+Copy the user ID you've received in the output into the following script and run it to remove the user from your Buildkite organization:  
+```bash
+mutation RemoveOrganizationMember {
+  organizationMemberDelete(input: { id: "user-ID-you-copied-goes-here" }) {
+    deletedOrganizationMemberID
+  }
+}
+```  
+
+The user is now removed from your Buildkite organization.
```
Comment (plaindocs) [inline pages/pipelines/permissions.md.erb:99]:
```suggestion
```

And I wouldn't bother with this.

File: pages/agent/v3/tokens.md.erb [hunk 5]
```diff
@@ -36,7 +36,9 @@ mutation {
 }
 ```
 
-The token description should clearly identify the environment the token is intended to be used for, and is shown on your [Agents page](https://buildkite.com/organizations/-/agents).
+The token description should clearly identify the environment the token is intended to be used for, and is shown on your [Agents page](https://buildkite.com/organizations/-/agents).
```
Comment (plaindocs) [inline pages/agent/v3/tokens.md.erb:39]: Maybe an example here?

`Read only token for the static site generator`

I dunno, maybe someone has a better idea?

**Review by plaindocs — COMMENTED** [review]
I can't wait to close off three docs tickets with one PR! :-) Good stuff.

A few suggested improvements inline.

File: pages/pipelines/permissions.md.erb (hunk 4 above)
Comment (karensawrey) [reply pages/pipelines/permissions.md.erb:99]: You work on a project that demands suuuuper granular extra details everywhere for two months and then you spend the next two years unlearning that, oops, sorry xD

File: pages/pipelines/permissions.md.erb (hunk 2 above)
Comment (karensawrey) [reply pages/pipelines/permissions.md.erb:70]: Oh, I'll reword that.

**Comment by karensawrey** [comment]
> I can't wait to close off three docs tickets with one PR! :-) Good stuff.
> 
> A few suggested improvements inline.

Yay. Awesome. I'll deal with the remaining edits ASAP.

File: pages/pipelines/permissions.md.erb [hunk 6]
```diff
@@ -87,7 +87,7 @@ query FindOrganizationMember {
 }
 ```  
 
-Copy the user ID you've received in the output into the following script and run it to remove the user from your Buildkite organization:  
+Copy the user ID you've received into the following script and run it to remove the user from your Buildkite organization:
```
Comment (plaindocs) [inline pages/pipelines/permissions.md.erb:90]: How about "into the following mutation" so we're not changing midway?

**Review by plaindocs — APPROVED** [review]
One tiny change, and let's merge this.

**Pull request merged**